src/main.rs

```diff
@@ -153,6 +203,19 @@ fn work() -> Result<(), Cue2CCDError> {
         return Err(Cue2CCDError::MissingFilesError { missing_files });
     }
 
+    let mut sbi_lba_array: Vec<i32> = Vec::new();
+    let mut sbi_data: Vec<Vec<u8>> = Vec::new();
+
+    // TODO: is this extension check case sensitive?
+    if Path::new(&output_stem.with_extension("sbi")).exists() {
+        // SBI files are very small, so it seems best to read the whole thing in first?
+        let result = generate_sbi_data(std::fs::read(Path::new(
```

Comment:
Rust has destructuring, which can simplify this a little: 
```suggestion
        let (sbi_lba_array, sbi_data) = generate_sbi_data(std::fs::read(Path::new(
```

Reply:
Ah, perfect, ty! That's much better